src/assets/MobileTestAutomation/automationFrameworksComparison.md

```diff
@@ -0,0 +1,162 @@
+
+Among the available frameworks, I've picked these five for comparison: Appium, Calabash, XCUITest, Espresso, and
+Selendroid. Let's break them down in straightforward.
+
+## [**Selendroid**](http://selendroid.io/)
+
+Selendroid, is a test automation framework for native or hybrid Android applications and the mobile web. It is an open
+source tool with 433 GitHub forks and 880 GitHub stars and [here](https://github.com/selendroid/selendroid) is the
+GitHub link.
+
+#### Pros and Cons:
+
+| **Pros 👍**                                                                                               | **Cons 👎**                                                                                                                  |
+|-----------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| **Environment:** Can be installed and run on various OS such as Windows, Linux, and macOS                 | **No Regular Updates:** Development has slowed down, which may limit its ability to support new Android features and devices |
+| **Open-Source:** Free to use but very less community for support                                          | **No Cross-Platform:** Limited to Android apps                                                                               |
+| **Hot Plugging** Selendroid supports hot plugging of hardware devices                                     | **Limited Language Support:** Primarily supports Java for writing tests                                                      |
+| **No App Modifications:** You don't need to recompile or modify your app                                  |                                                                                                                              |
+| **Inspector Tools:** Selendroid includes its own [UI inspector](http://selendroid.io/inspector.html) tool |                                                                                                                              |
+
+#### When to use?
+
+- When automating tests for Android applications across devices with varying screen sizes and resolutions
+- For conducting cross-browser testing of Android apps, ensuring compatibility across a wide range of devices and
+  Android versions
+- When you require simultaneous testing across multiple Android devices, enhancing overall test efficiency
+- When testing scenarios that require intricate gestures and interactions, benefiting from its strong capabilities in
+  handling touch events
+- If seamless integration with continuous integration tools is needed, enabling automated testing within a CI/CD
+  pipeline
+
+## [**Espresso**](https://developer.android.com/training/testing/espresso)
+
+Espresso is an open source Android applications test automation framework developed by Google. It's known for its
```

Comment:
since it's open source, maybe add GitHub link too like for the one above?

Reply:
I couldn't find the repo for Espresso.

Reply:
@grgprarup what about this one? It looks like it's part of android test
https://github.com/android/android-test/tree/main/espresso

Reply:
Thanks for the link. I have also added the GitHub link in the content and the docs link in the title.